app/models/accounts.rb

```diff
@@ -0,0 +1,4 @@
+class Account < ApplicationRecord
+  has_many :oders, dependent: :destroy
+  has_many :address, dependent: :destroy
```

Comment:
has_many thì em để tên collection là số nhiều nhé
```suggestion
  has_many :addresses, dependent: :destroy
```